### PR TITLE
[DuckDuckGo] Update targets

### PR DESCRIPTION
**Pull request by coffeemakr**
Many certificates have been fixed. Also Include `favicons.` and `icons.` with the correct exclusion rule and tests.
`va-l1.duckduckgo.com` doesn't seem to exist anymore.

### CODE REVIEW

File: src/chrome/content/rules/DuckDuckGo.xml [hunk 1]
```diff
@@ -41,24 +34,35 @@
...
+	<target host="favicons.duckduckgo.com" />
+		<test url="http://favicons.duckduckgo.com/ip2/twitter.com.ico" />
+		<exclusion pattern="^http://favicons\.duckduckgo\.com/$" />
 	<target host="ff.duckduckgo.com" />
+	<target host="help.duckduckgo.com" />
 	<target host="i.duckduckgo.com" />
 	<target host="images.duckduckgo.com" />
+	<target host="icons.duckduckgo.com" />
+		<test url="http://icons.duckduckgo.com/ip2/twitter.com.ico" />
+		<exclusion pattern="^http://icons\.duckduckgo\.com/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/DuckDuckGo.xml]: Why is the exclusion here required?

File: src/chrome/content/rules/DuckDuckGo.xml [hunk 2]
```diff
@@ -20,17 +20,10 @@
...
-			 - blog.duckduckgo.com
-			 - help.duckduckgo.com
-			 - meme.duckduckgo.com
-			 - settings.duckduckgo.com
-			 - shop.duckduckgo.com
-			 - wiki.duckduckgo.com
 			 - www.ddg.gg
 
-		5xx server error:
-			 - favicons.duckduckgo.com
```
Comment (J0WI) [inline src/chrome/content/rules/DuckDuckGo.xml]: You may keep this somewhere around, that we know why the exclusion is there.

File: src/chrome/content/rules/DuckDuckGo.xml (hunk 1 above)
Comment (coffeemakr) [reply src/chrome/content/rules/DuckDuckGo.xml]: Strange. When I tried the two domains yesterday it resulted in a 500 error. Know it shows an search interface which results in a 502 when used to search. :/

**Comment by coffeemakr** [comment]
I have removed the comment and the exception because the index pages for `favions` and `icons` don't deliver a 5xx error anymore.

**Pull request merged**